### PR TITLE
Generalize usage of IsVessel

### PR DESCRIPTION
**Pull request by TheGondos**
nassputils.h defines an IsVessel helper to test for a vessel class, use it where applicable.
This commits adds support for ML, Crawler, MainChute and DrogueChute to it.
This fixes lm_telecom.cpp where it tested only for "ProjectApollo\\Saturn5"

### CODE REVIEW

File: Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj [hunk 1]
```diff
@@ -96,7 +96,7 @@
       <AssemblerListingLocation>.\Release\LCC/</AssemblerListingLocation>
       <ObjectFileName>.\Release\LCC/</ObjectFileName>
       <ProgramDataBaseFileName>.\Release\LCC/</ProgramDataBaseFileName>
-      <AdditionalIncludeDirectories>../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project;</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\github\NASSP\Orbitersdk\samples\ProjectApollo\src_aux;../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project</AdditionalIncludeDirectories>
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj]: This is a path on your computer only.

File: Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj [hunk 2]
```diff
@@ -127,7 +127,7 @@
       <AssemblerListingLocation>.\Debug\LCC/</AssemblerListingLocation>
       <ObjectFileName>.\Debug\LCC/</ObjectFileName>
       <ProgramDataBaseFileName>.\Debug\LCC/</ProgramDataBaseFileName>
-      <AdditionalIncludeDirectories>../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project;</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\github\NASSP\Orbitersdk\samples\ProjectApollo\src_aux;../../src_rtccmfd;../../src_sys;../../../../include;../../../../../Sound/OrbiterSound_SDK/VESSELSOUND_SDK/ShuttlePB_project</AdditionalIncludeDirectories>
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj]: Idem

**Review by ThymoNL — CHANGES_REQUESTED** [review]

File: Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj (hunk 2 above)
Comment (TheGondos) [reply Orbitersdk/samples/ProjectApollo/Build/VC2017/LCC.vcxproj]: Nice job Visual Studio, the only thing I did through the GUI and the default behavior is crap...
At least now I know, I will be more careful in the future...

**Comment by indy91** [comment]
@ThymoNL you never approved the changes that @TheGondos made with the include path. I would like to merge this if there were no other issues.

**Comment by ThymoNL** [comment]
Whoops. This obviously slipped from my attention. LGTM. Approved :)

**Review by ThymoNL — APPROVED** [review]

**Pull request merged**